here-naksha-app-service/src/test/java/com/here/naksha/app/service/NakshaAppTest.java

```diff
@@ -124,25 +124,26 @@ void tc0002_testCreateDuplicateStorage() throws Exception {
   }
 
   @Test
-  @Order(2)
-  void tc0003_testGetStorages() throws Exception {
-    // Test API : GET /hub/storages
+  @Order(3)
+  void tc0003_testCreateStorageMissingClassName() throws Exception {
+    // Test API : POST /hub/storages
```

Comment:
I now figured most of these comments are wrong due to copy pasting, will refactor

Reply:
done